redisinsight/electron/index.html

```diff
@@ -19,16 +19,16 @@
 <body class="electron">
   <div id="root" style="height: 100%; width: 100%;"></div>
   <script>
-    if (typeof process === 'object') {
+    if (typeof envVars === 'object') {
       const scripts = [];
 
-      if (process.env.NODE_ENV === 'development') {
+      if (envVars.NODE_ENV === 'development') {
         // Dynamically insert the DLL script in development env in the
         // renderer process
         scripts.push('../dll/renderer.dev.dll.js');
 
         // Dynamically insert the bundled app script in the renderer process
-        const port = process.env.PORT || 1212;
+        const port = envVars.PORT || 1212;
```

Comment:
again if we decide to use envVars - let's implement it as config on BE.  when you have all default values inside it and here just get the value